packages/vuetify/src/labs/VDataTable/composables/items.ts

```diff
@@ -11,26 +11,30 @@ export interface DataTableItemProps {
   items: any[]
   itemValue: SelectItemKey
   returnObject: boolean
+  itemKey: SelectItemKey
```

Comment:
Here I added the new prop for having different variables to get the important values